test/specs/mainnet/swap/Smoke_CrossChainQuote.spec.js

```diff
@@ -0,0 +1,4195 @@
+          "The Data value of the Batch Execution Account Transaction is not displayed."
+        );
+      } catch (e) {
+        console.log(e);
+      }
+
+      // Batch the cross chain transaction
+      let { to, value, data } = quote.transaction;
+      batchCrossChainTransaction =
+        await sdkMainNet.batchExecuteAccountTransaction({
+          to,
+          data: data,
+          value,
+        });
+
+      // console.log( "gateway batch transfer token transaction::: ",batchCrossChainTransaction);
+    }
+
+    for (let j = 0; j < batchCrossChainTransaction.requests.length; j++) {
+      try {
+        assert.isNotEmpty(
+          batchCrossChainTransaction.requests[j].to,
+          "The To value of the Batch Execution Account Transaction is not displayed."
+        );
+      } catch (e) {
+        console.log(e);
+      }
+
+      try {
+        assert.isNotEmpty(
+          batchCrossChainTransaction.requests[j].data,
+          "The Data value of the Batch Execution Account Transaction is not displayed."
+        );
+      } catch (e) {
+        console.log(e);
+      }
+    }
+
+    try {
+      assert.isNull(
+        batchCrossChainTransaction.estimation,
+        "The Data value of the Batch Execution Account Transaction is not displayed."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    // Estimating the batch
+    let EstimatedGas_Estimate;
+    let FeeAmount_Estimate;
+    let EstimatedGasPrice_Estimate;
+
+    let estimationResponse = await sdkMainNet.estimateGatewayBatch();
+    // console.log("Gas Estimation::: ", estimationResponse);
+
+    for (let k = 0; k < estimationResponse.requests.length; k++) {
+      try {
+        assert.isNotEmpty(
+          estimationResponse.requests[k].to,
+          "The To value of the Batch Execution Account Transaction is not displayed."
+        );
+      } catch (e) {
+        console.log(e);
+      }
+
+      try {
+        assert.isNotEmpty(
+          estimationResponse.requests[k].data,
+          "The Data value of the Batch Execution Account Transaction is not displayed."
+        );
+      } catch (e) {
+        console.log(e);
+      }
+    }
+
+    try {
+      assert.isNotEmpty(
+        estimationResponse.estimation.feeAmount,
+        "The value of the feeAmount field of the Estimation Response is not displayed."
+      );
+      FeeAmount_Estimate = estimationResponse.estimation.feeAmount._hex;
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.strictEqual(
+        estimationResponse.estimation.feeTokenReceiver,
+        "0xf593D35cA402c097e57813bCC6BCAb4b71A597cC",
+        "The feeTokenReceiver Address of the Estimate Batch Response is not displayed correctly."
+      );
+    } catch (e) {
+      console.log(e);
+    }
+
+    try {
+      assert.isNumber(
+        estimationResponse.estimation.estimatedGas,
+        "The estimatedGas of the Estimate Batch Response is not displayed."
```

Comment:
Remember to change the assertion message to reference the issue that it is not a numbber

Reply:
Updated the texts from all the files.